site/viamguides/src/environment-variables/environment-variables.md

```diff
@@ -0,0 +1,223 @@
+    ```bash
+    pip3 install python-dotenv
+   ```
+1. **Load the variables into your environment**: At the top of your `square.py` file, add the following code to load variables into your environment.
+    ```python
+    from dotenv import load_dotenv
+    import os
+
+    load_dotenv()  # Loads the environment variables from the .env file
+
+    robot_api_key = os.getenv('ROBOT_API_KEY')
+    robot_api_key_id = os.getenv('ROBOT_API_KEY_ID')
+    ```
+1. **Use variables in your code**: You can now use these variables within your code. On rows 7 and 8 in `square.py`, you load variables into the environment. And then on rows 20 and 21, you use those variables to access your machine.
+  ![Code in VS Code](assets/vscode.png)
+1. **Run the code:** Run the sample code to connect to your machine
+   ```bash
+   python3 square.py
+   ```
+
+If you want to play with the rover, go to the Viam app **CONTROL** tab to interact with your machine’s resources. Click on one of the camera panels and toggle the camera stream on so you can observe the rover’s movements. Or try adding more code to [drive the rover in a square](https://docs.viam.com/get-started/drive-rover/).
+
+In the next few sections, learn how to use variables on your machine in production.
+
+<!-- ------------------------ -->
+## Add the control code to your machine
+Duration: 3
+
+Once you have the code working in your local development environment, you may choose to move your control code to your machine, so that it can run in production. 
+
+> aside positive
+> If you followed [Try Viam](https://app.viam.com/try), you won't have access to add control code to the machine, since you are just borrowing the rover. However, you can skip to the next part to see how you can set up variables on the machine. If you're using your own wheeled robot, you can complete these steps.
+
+### Add the control code to your machine
+1. **SSH into your board**: From the terminal window, run the following command to [SSH (Secure Shell) into your board](https://docs.viam.com/installation/prepare/rpi-setup/#connect-with-ssh), where the text in `<>` should be replaced (including the `<` and `>` symbols themselves) with the `user` and `hostname` you configured when you set up your machine. 
+    ```bash
+    ssh <USERNAME>@<REMOTE-HOSTNAME>.local
+    ```
+1. **Create a project directory**: Create a new directory on your machine to hold your project code.
+    ```bash
+    mkdir robot
+    ```
+1. **Install dependencies**: Install the Viam Python SDK (and other dependencies if required) into the folder. 
+    ```bash
+    pip3 install viam-sdk
+    ```
+<!-- TODO do i need to activate a venv? -->
+<!-- TODO do i need to chmod? -->
+1. **Copy file(s) to the machine**: From the original computer you were developing on (not the secure shell prompt accessing your machine), copy the `square.py` file to your machine.
+    ```bash
+    scp square.py <USERNAME>@<REMOTE-HOSTNAME>.local:/home/myboard/robot/square.py
```

Comment:
The default directory for scp is the same as when ssh-ing into a machine, which is the home directory of the username logging in. This means you can use a relative path after the `:` to make the command more copy-pastable:

```suggestion
    scp square.py <USERNAME>@<REMOTE-HOSTNAME>.local:./robot/square.py
```